micro_dl/cli/torch_inference_script.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
Tiny nitpicky detail, but it's nice to sort imports alphabetically, external imports first then those from the repo. It helps you see quicker what's there.